### PR TITLE
Partial cleanup of issue #595   Part2

### PR DESCRIPTION
**Pull request by afabri**
Some more things to do.

### CODE REVIEW

**Comment by lrineau** [comment]
Will be tested in [CGAL-4.9-Ic-4][4]

[4]: https://cgal.geometryfactory.com/CGAL/testsuite/results-4.9-Ic-4.shtml

File: Kinetic_data_structures/include/CGAL/Polynomial/basic.h [hunk 1]
```diff
@@ -203,9 +203,5 @@ struct Bezier_tag {};
 } } //namespace CGAL::POLYNOMIAL
 */
 
-#ifdef CGAL_POLYNOMIAL_NO_LIMITS
-#include <CGAL/Polynomial/internal/limits.h>
-#else
 #include <limits>
-#endif
-#endif
+
```
Comment (lrineau) [inline Kinetic_data_structures/include/CGAL/Polynomial/basic.h]: The last `#endif` must not be removed!

**Comment by afabri** [comment]
please retest

**Comment by lrineau** [comment]
Will be tested in [CGAL-4.9-Ic-5][5]

[5]: https://cgal.geometryfactory.com/CGAL/testsuite/results-4.9-Ic-4.shtml

**Comment by lrineau** [comment]
Is the issue #595 solved?

**Pull request merged**